### PR TITLE
add waiting time for customers

### PR DESCRIPTION
**Pull request by KarishmaSinghChauhan**

### CODE REVIEW

File: frontendservice/src/views/specificShop.vue [hunk 1]
```diff
@@ -22,6 +22,7 @@ specific language governing permissions and limitations under the License. */
           v-else
           class="title"
         >Number of people currently in queue are {{shopinfo.customersInQueue}}</p>
+        <p v-if="totalWaitingTime != 0" class="title">Your waiting time is: {{totalWaitingTime}} mins</p>
```
Comment (priytanvani) [inline frontendservice/src/views/specificShop.vue:25]: Nit. Change this to "Approx. waiting time:" ?

Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/specificShop.vue:25]: The variable name should be approxWaitingTime you mean? Or talking about the message displayed in the UI?

File: frontendservice/src/views/specificShop.vue [hunk 2]
```diff
@@ -142,6 +143,25 @@ export default {
             });
           }
         });
+
+        self.getWaitingTime();
+
+        if(this.flag) {
```
Comment (bavanya) [inline frontendservice/src/views/specificShop.vue:149]: karishma please change the flag names in your pr as per the update-flag-names pr i just sent out.

Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/specificShop.vue:149]: I pulled the code from the master and implemented the waiting time. Can you merge your PR after my PR gets merged please if there's anything like that?

Comment (bavanya) [reply frontendservice/src/views/specificShop.vue:149]: karishma pull from master and change the flag name here as per it.

File: frontendservice/src/views/specificShop.vue (hunk 1 above)
Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/specificShop.vue]: Changed.

File: frontendservice/src/views/specificShop.vue (hunk 2 above)
Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/specificShop.vue]: Done. Please check once.

Comment (bavanya) [reply frontendservice/src/views/specificShop.vue]: yes all is good :)

File: frontendservice/src/views/specificShop.vue [hunk 3]
```diff
@@ -142,6 +143,25 @@ export default {
             });
           }
         });
+
+        self.getWaitingTime();
+
+        if(this.isTokenInCookie) {
+          this.totalWaitingTime = this.waitingTimePerCustomer * this.tokeninfo.customersAhead;
+        } else{
```
Comment (sdumka) [inline frontendservice/src/views/specificShop.vue:151]: add space after else

File: frontendservice/src/views/specificShop.vue [hunk 4]
```diff
@@ -142,6 +143,25 @@ export default {
             });
           }
         });
+
+        self.getWaitingTime();
+
+        if(this.isTokenInCookie) {
```
Comment (sdumka) [inline frontendservice/src/views/specificShop.vue:149]: space after if

File: frontendservice/src/views/specificShop.vue [hunk 5]
```diff
@@ -142,6 +143,25 @@ export default {
             });
           }
         });
+
```
Comment (sdumka) [inline frontendservice/src/views/specificShop.vue:146]: remove new line

File: frontendservice/src/views/specificShop.vue [hunk 6]
```diff
@@ -142,6 +143,25 @@ export default {
...
+        self.getWaitingTime();
+
```
Comment (sdumka) [inline frontendservice/src/views/specificShop.vue:148]: remove new line

File: frontendservice/src/views/specificShop.vue [hunk 7]
```diff
@@ -142,6 +143,25 @@ export default {
...
+        if(this.isTokenInCookie) {
+          this.totalWaitingTime = this.waitingTimePerCustomer * this.tokeninfo.customersAhead;
+        } else{
+          this.totalWaitingTime = this.waitingTimePerCustomer * this.shopinfo.customersInQueue;
+        }
+    },
+
```
Comment (sdumka) [inline frontendservice/src/views/specificShop.vue:155]: remove extra line.. here and at other places

File: frontendservice/src/views/specificShop.vue (hunk 3 above)
Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/specificShop.vue]: Done.

File: frontendservice/src/views/specificShop.vue (hunk 4 above)
Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/specificShop.vue]: Done.

File: frontendservice/src/views/specificShop.vue (hunk 5 above)
Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/specificShop.vue]: Done.

File: frontendservice/src/views/specificShop.vue (hunk 6 above)
Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/specificShop.vue]: Done.

File: frontendservice/src/views/specificShop.vue (hunk 7 above)
Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/specificShop.vue]: Done.

File: frontendservice/src/views/specificShop.vue [hunk 8]
```diff
@@ -142,8 +143,22 @@ export default {
...
           }
         });
+        self.getWaitingTime();
+        if (this.isTokenInCookie) {
+          this.totalWaitingTime = this.waitingTimePerCustomer * this.tokeninfo.customersAhead;
+        } else {
+          this.totalWaitingTime = this.waitingTimePerCustomer * this.shopinfo.customersInQueue;
+        }
+    },
+    getWaitingTime() {
```
Comment (sdumka) [inline frontendservice/src/views/specificShop.vue:153]: getTotalWaitingTime?

File: frontendservice/src/views/specificShop.vue [hunk 9]
```diff
@@ -178,17 +192,15 @@ export default {
...
-
     beforeDestroy() {
       clearInterval(this.timer);
     }
   },
-
   created() {
+    this.getWaitingTime();
     this.getShopInfo();
     this.timer = setInterval(this.getShopInfo, 1 /*1 second */);
```
Comment (sdumka) [inline frontendservice/src/views/specificShop.vue:205]: Unrelated but Add space before 1

Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/specificShop.vue]: Done.

File: frontendservice/src/views/specificShop.vue (hunk 8 above)
Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/specificShop.vue]: Changed to getWaitingTimePerCustomer() as discussed.

**Pull request merged**